index.html

```diff
@@ -26,6 +26,7 @@
       }
     </style>
   </head>
+  <link rel="stylesheet" href="css/index.css"></link rel>
```

Comment:
```suggestion
  <link rel="stylesheet" href="css/index.css">
```

the `</link rel>` at the end isnt actually doing anything, take it off and see! 

a "link" tag is one type of "self-closing tag" in html. As a general rule: If you're writing stuff inside the chevrons (<>) of that tag, its a self closing html tag! 

Example: link tags are self closing, and title tags are not. Notice how the content of a link tag is written inside the tag `<>` chevrons itself:  `<link rel="stylesheet" href="styles.css">`

But the title you want to display in a title tag is written between two sets of chevrons that mark the start and end of the title: 
```html
<title>
HTML Tags
</title>
```